_templates/component/new/add-dynamic-export.ejs.t

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
we'll need to distinguish between blocks and components. additionally, we wouldn't want blocks in the `component` package. they'd be scoped to the `web` app